pkgs/development/libraries/polkit/default.nix

```diff
@@ -58,6 +57,12 @@ stdenv.mkDerivation rec {
       url = "https://gitlab.freedesktop.org/polkit/polkit/-/commit/7ba07551dfcd4ef9a87b8f0d9eb8b91fabcb41b3.patch";
       sha256 = "ebbLILncq1hAZTBMsLm+vDGw6j0iQ0crGyhzyLZQgKA=";
     })
+
+    # FIXME: Fixes build for version 124, remove after next release
```

Comment:
Please add a link to the upstream PR to the comment.

Btw, you should be able to use `PKG_CONFIG_SYSTEMD_SYSUSERS_DIR` environment variable to override it as well.

Reply:
Thanks for reviewing, will do.

Huh, where is this variable from? Is this just a pkg-config feature I don't know about? I can find `PKG_CONFIG_SYSTEMD_SYSUSERSDIR` used in gnome-initial-setup in nixpkgs but don't know where this is from.

Reply:
It is [quite recent feature](https://gitlab.freedesktop.org/pkg-config/pkg-config/-/commit/ae0a8b1ad5e40f68fc09ffedac215e038a50f070#124c0becfe68b1ef671f49ed2b9d24779ace126f_341_341). Most online manpages do not seem to know it, even thought it was introduced eight years ago.

Reply:
Another question about the other patch that is pulled from gitlab.freedesktop.org: Should this be pulled into the repo locally? I don't think that the gitlab instance is going anywhere in the next while but maybe the archived project could be removed eventually? (I don't know any plans from upstream)

Reply:
Yeah, good idea.

Reply:
Upstream didn't like the patch, so I instead followed you suggestion and used the switched to the pkg-config environment variables to work around the broken meson option. Think this might ready now. :)